layouts/partials/block/data.html

```diff
@@ -41,37 +42,44 @@
   ======================
 */}}
 
-{{ if "github" | in $src }}
-  {{/* Change base URL to api.github.com */}}
-  {{ $newSrc := replace $src "https://github.com/CodeYourFuture/" "https://api.github.com/repos/CodeYourFuture/" }}
+{{ if or "png" "webp" "jpg" "jpeg" "gif" | in $src }}
```

Comment:
I'm increasingly feeling like we maybe want an explicit "block type" tag in blocks, rather than inferring from what the URL looks like... WDYT?

Reply:
I think about this from time to time. I just really appreciate the lower friction from having this partial do it for me.